distributions/openhab/src/main/filtered-resources/userdata/etc/version.properties

```diff
@@ -7,6 +7,5 @@ Repository        Version
 ----------------------------------------
 openhab-distro  : ${project.version}
 openhab-core    : ${ohc.version}
-openhab1-addons : ${oh1.version}
-openhab2-addons : ${oh2.version}
+openhab-addons  : ${oh2.version}
```

Comment:
`oh2.version` does not seem to be appropriate anymore... How about changing it to `oha.version`?

Reply:
I thought of using that abbreviation too. But I didn't update it because it's also used by the pipeline.

Reply:
Ok, then let's leave that to a follow-up PR.